src/plugins/views/Tabs/hooks/useTabLayout.js

```diff
@@ -140,9 +140,8 @@ const useTabLayout = (props, dockRef) => {
       let tabExists = maxboxChildren?.tabs.find((t) => t.id === layoutActiveId);
 
       if (!tabExists) {
-        tabExists = _layout.dockbox.children[0].tabs.find(
-          (t) => t.id === layoutActiveId,
-        );
+        if (maxboxChildren?.tabs)
+          tabExists = maxboxChildren.tabs.find((t) => t.id === layoutActiveId);
       }
       activeTabId.current = layoutActiveId;
```

Comment:
I looked at this for some time, and this makes 0 sense. We're NEVER checking `dockbox` anymore for the tab. The first approach you did, seemed to work correctly. This is just nonsense. line 144 that you introduced is exactly the same as line 140. Where it used to be a check inside `dockbox`, not `maxbox` again.

Reply:
Ok. In this case you are correct.

Reply:
Testing it, it just works. Maybe that's due to the fact that the maxbox and the dockbox children are the same.
Do you have any reason to think of a situation where they aren't?

Reply:
Testing what? Did you test every possible scenario? If you did and nothing broke, we're good to go. As I doubt that happened, my request changes still mantains. I know I added that a few years ago because of an edge case scenario. Approving this code as is will fix YOUR issue and cause at least 1 regression.
Before you ask, I don't remember what, but if you dig a bit you'll find the ticket that first originated that code and then you can test it.